recorder/pom.xml

```diff
@@ -135,7 +135,7 @@
         <profile>
             <id>native</id>
             <activation>
-                <activeByDefault>true</activeByDefault>
+                <activeByDefault>false</activeByDefault>
```

Comment:
This will lead to test failures. Because recorder is eventually going to have tests that expect jvmti agent to be present and will run JVMs with it.

Why did this have to be disabled? Did something conflict with this profile?